luigi/contrib/k8s_job.py

```diff
@@ -186,8 +216,14 @@ def run(self):
                 }
             }
         }
+        job_json['metadata']['labels'].update(self.labels)
         job = Job(self.__kube_api, job_json)
         job.create()
         # Track the Job (wait while active)
         self.__logger.info("Start tracking Kubernetes Job: " + self.uu_name)
         self.__track_job()
+
+    def output(self):
```

Comment:
I'll just replace this with a pass as well and leave the pseudo code for what I would implement it as in combination with the potential `signal_complete`.